src/test/java/org/mockitousage/annotation/MockInjectionUsingSetterOrPropertyTest.java

```diff
@@ -50,10 +51,19 @@
 
     @Spy private TreeSet<String> searchTree = new TreeSet<String>();
 
+    private AutoCloseable session;
+
     @Before
     public void enforces_new_instances() {
-        // initMocks called in TestBase Before method, so instances are not the same
-        MockitoAnnotations.openMocks(this);
+        // openMocks called in TestBase Before method, so instances are not the same
+        session = MockitoAnnotations.openMocks(this);
```

Comment:
We should update `openMocks` to use `@CheckReturnValue`. Let's fix that in a separate PR.

Reply:
+1